Source/DedicatedServer/Private/ServerConsoleLinux.cpp

```diff
@@ -1,23 +1,244 @@
 // Copyright 2004-2016 YaS-Online, Inc. All Rights Reserved.
 
 #include "DedicatedServerPrivatePCH.h"
+#if PLATFORM_LINUX
+#include <iostream>
+#include <sstream>
+#include <string>
+#endif
 
 void DumpConsoleHelp();
 
 #if WITH_SERVER_CODE && PLATFORM_LINUX
 	void FServerConsole::Tick()
 	{
+		int hInputEvents = getch();
+		if ( hInputEvents != ERR )
+		{
+			if ( hInputEvents == '\n' )
+			{
+				auto& sInput = m_sInput;
+				AsyncTask( ENamedThreads::GameThread, [this, sInput]()
+				{
+					TCHAR sOutput[MAX_SPRINTF] = TEXT("");
```

Comment:
```suggestion
					TCHAR sOutput[MAX_SPRINTF] = TEXT( "" );
```

PS: you're fine to ignore the suggestions too and fix it when you push changes, it is usually just easier to highlight the diff than to explain what should be done